trillian_log_api.proto

```diff
@@ -99,6 +99,25 @@ message QueueLeavesResponse {
     repeated QueuedLogLeaf queued_leaves = 2;
 }
 
+message AddSequencedLeafRequest {
+    int64 log_id = 1;
+    LogLeaf leaf = 2;
+}
+
+message AddSequencedLeafResponse {
+    google.rpc.Status status = 2;
+}
+
+message AddSequencedLeavesRequest {
+    int64 log_id = 1;
+    repeated LogLeaf leaves = 2;
+}
+
+message AddSequencedLeavesResponse {
+    // Same number and order as in the corresponding request.
```

Comment:
Unfortunately it's bad practice to rely on any ordering coming back from the server. 
Consider putting status inside `LogLeaf` 

See https://cloud.google.com/apis/design/

Reply:
See also [`QueuedLogLeaf`](https://github.com/google/trillian/blob/master/trillian_log_api.proto#L75) for something that bundles a leaf+status together.

Reply:
Yup, ended up using `QueuedLogLeaf`. I leave the ordering comment as is to be consistent with `QueueLeaves` which does it as well.